modules/oe_content_featured_media_field/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenEuropa Content Featured Media Field
+
+This module provides the Featured media field type containing a media reference field and a caption textfield.
+It includes an autocomplete widget, and an entity browser widget for the field type.
+
+## Installation
+
+Before enabling this module, make sure that the following module is present in your codebase by adding it to your
```

Comment:
The dependency is only needed if you want to use the entity browser based widget.